test/test_kubeclient.rb

```diff
@@ -368,17 +340,19 @@ def test_api_bearer_token_with_params_success
 
   def test_api_bearer_token_success
     stub_request(:get, %r{/api/v1$})
-      .to_return(body: open_test_file('core_api_resource_list.json'),
-                 status: 200)
+      .to_return(
+        body: open_test_file('core_api_resource_list.json'), status: 200
+      )
```

Comment:
I must say I find the previous style in this particular case more readable.
2 lines became 3 but the actual arguments got crammed on one line.

IMHO the natural progression as lines get too long and require splitting is from:
```
func(arg1, arg2)
```
to
```
func(long_argument1,
     long_argument2)
```
to
```
a_long.function_call(
  long_argument1,
  long_argument2
)
```

IIUC many of your changes are forced by Rubocop forbidding the 2nd option, so you compacted to 1st or expanded to 3rd, right?
I don't feel strongly about this (and I like most of your changes!) but I wonder if *enforcing* this is beneficial.  Is there a Rubocop setting that would make it accept the 2nd form, leaving it to human discretion?